### PR TITLE
DBR-153 Language selector component

### PR DESCRIPTION
**Pull request by damianmuti**
Created a simple component to handle language selector.

### CODE REVIEW

File: src/components/Signup/Signup.js [hunk 1]
```diff
@@ -99,27 +100,7 @@ export default injectIntl(function({ intl }) {
...
-                <span>
-                  <strong>EN</strong>
-                </span>
-                <Link to="?lang=es" className="option--item">
-                  ES
-                </Link>
-              </div>
-            )}
-          </div>
+          <LanguageSelector lang={intl.locale} />
```
Comment (andresmoschini) [inline src/components/Signup/Signup.js]: I am not sure, is just an opinion, what do you think calling it `currentLang`?

File: src/components/shared/LanguageSelector/LanguageSelector.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+const LanguageSelector = ({ lang }) => {
```
Comment (andresmoschini) [inline src/components/shared/LanguageSelector/LanguageSelector.js]: We could inject current lang in place of receiving it as a parameter. But I am not sure if it is a good idea, @fcoronelmakingsense, @damianmuti  what do you think?

```
/**
 * LanguageSelector
 * @param { Object } props
 * @param { import('react-intl').InjectedIntl } props.intl
 */
const LanguageSelector = ({ intl }) => {
  const lang = intl.locale;
  return (
    <div className="language-selector">
    // . . .
    </div>
  );
};
export default injectIntl(LanguageSelector);
```

File: src/components/Signup/Signup.js (hunk 1 above)
Comment (damianmuti) [reply src/components/Signup/Signup.js]: I like that approach, @andresmoschini . Will do the proper changes 👍

File: src/components/shared/LanguageSelector/LanguageSelector.js (hunk 2 above)
Comment (fcoronelmakingsense) [reply src/components/shared/LanguageSelector/LanguageSelector.js]: always has the current language, and that is how we use in other pages. maybe is better this

Comment (damianmuti) [reply src/components/shared/LanguageSelector/LanguageSelector.js]: This was my first thought when I've created this component, but my React knowledge isn't the best tho, so I couldn't get around it.
Thanks for the help, @andresmoschini ! I got it working this way, and I guess it's a good idea not to not have the locale as a parameter. 
@fcoronelmakingsense do you agree? If it's ok with you guys, I'll push the changes right away.

Comment (andresmoschini) [reply src/components/shared/LanguageSelector/LanguageSelector.js]: 🚀

Comment (damianmuti) [reply src/components/shared/LanguageSelector/LanguageSelector.js]: I'll update this PR right away 👍

**Comment by andresmoschini** [comment]
:shipit:

**Pull request merged**